USING.md

```diff
@@ -0,0 +1,56 @@
+
+## What is OPAM
+
+OPAM is the package manager for the OCaml programming language, the language 
+in which Coq is implemented.
+Instructions on
+[how to install OPAM](http://opam.ocaml.org/doc/Install.html)
+itself are available on the OPAM website.
+
+By following the instructions on this web page one installs the last stable
+version of Coq and all additional packages in the directory `~/opam-coq.@COQV@`.
+Instructions target an OPAM newcomer.
+
+## Using OPAM to install Coq
+
+Once the `opam` command is available, i.e. OPAM is installed, one can
+proceed as follows:
+
+    export OPAMROOT=~/opam-coq.@COQV@ OPAMJOBS=2
+    opam init -n --comp=@OCAMLV@
+    eval `opam config env`
```

Comment:
Since the only commands you execute afterward are prefixed by `opam`, you don't need to modify the environment.